rdf4j/src/test/java/org/apache/commons/rdf/rdf4j/NativeStoreGraphTest.java

```diff
@@ -49,17 +51,23 @@
  */
 public class NativeStoreGraphTest extends AbstractGraphTest {
 
-	public final class NativeStoreFactory implements RDFTermFactory {
+	public final class NativeStoreRDF implements RDF {
```

Comment:
Just to satisfy `AbstractGraphTest` which blindly calls `createGraph()` on the `RDF` factory. The functionality is also there in `RDF4J` using `asGraph`.   Perhaps a richer `NativeStore` factory could be made that hides the `ValueFactory` plumbing and can be created with a folder `Path`?  

Unclear if such a factory would then reuse the same connection for each `createGraph()` and not support `createDataset()` (more than once), or if it would just return the populated Dataset, or if each call would make a new connection in a new folder.. 

Let's discuss it in [COMMONSRDF-48](https://issues.apache.org/jira/browse/COMMONSRDF-48) as I think that would be generally useful to have a shortcut. (and as well for Jena backed by TDB)